appstore_tools/actions/download.py

```diff
@@ -21,12 +18,12 @@
 
 
 def download_version(
-    access_token: str,
+    access_token: AccessToken,
     app_dir: str,
     app_id: str,
     bundle_id: str,
     platforms: appstore.PlatformList,
-    version_states: appstore.VersionStateList = list(appstore.VersionState),
+    version_states: appstore.VersionStateList = tuple(appstore.VersionState),
```

Comment:
If we are changing the default value to a Tuple, we need to change it in line 37 as well (and similar places).

Reply:
probably it doesn't fit this PR but in general you shouldn't have mutable defaults in functions in python (see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Reply:
(and lists are immutable)

Reply:
modified everywhere in the file now :) I see other places where you used lists in defaults, but for now I wouldn't change them

Reply:
thanks for the fix! yeah, it's off topic for the PR, but no problem leaving it in here.

Reply:
> probably it doesn't fit this PR but in general you shouldn't have mutable defaults in functions in python (see https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments)

Good read.  thanks for the link.